crates/iota-data-ingestion-core/src/tests.rs

```diff
@@ -153,6 +153,30 @@ impl Reducer<TestWorker> for FixedBatchSizeReducer {
     }
 }
 
+/// This reducer implementation always returns an error when committing a batch.
+///
+/// Useful for testing graceful shutdown logic and behaviours
```

Comment:
looks like the end of sentence is missing, what behaviours?

Reply:
fixed: https://github.com/iotaledger/iota/pull/5820/commits/da658404e6778cfac7c2fc5de93126cce9b81cd0#diff-778f3f501ae6d64e48ac4f94eb52f1724d5c237e4b096697a9683bfd062ba8acR158